swap_meet/decor.py

```diff
@@ -1,2 +1,11 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, id = None, width = 0, length = 0, condition = 0, age = 0):
+        super().__init__(id, condition, age)    # Call the __init__() method of the parent class (Item) to set the ID attribute
+        self.width = width      # Set the width attribute of the Decor object
+        self.length = length    # Set the length attribute of the Decor object
+    def __str__(self):
```

Comment:
Space between functions

```suggestion
        self.length = length    # Set the length attribute of the Decor object

    def __str__(self):
```